src/to_sql.rs

```diff
@@ -6,6 +6,44 @@ use std::borrow::Cow;
 use uuid::Uuid;
 
 /// A conversion trait to a TDS type.
+///
+/// An implementation to a Rust type is needed for using it as a parameter in
+/// the [`Client#query`] or [`Client#execute`] methods. The following Rust types
+/// are already implemented to match the given server types:
+///
+/// |Rust type|Server type|
+/// |--------|--------|
+/// |`i8`|`tinyint`|
+/// |`i16`|`smallint`|
+/// |`i32`|`int`|
+/// |`i64`|`bigint`|
+/// |`f32`|`float(24)`|
+/// |`f64`|`float(53)`|
+/// |`bool`|`bit`|
+/// |`String`/`&str` (< 4000 characters)|`nvarchar(4000)`|
+/// |`String`/`&str`|`nvarchar(max)`|
+/// |`Uuid`|`uniqueidentifier`|
+/// |`Vec<u8>`/`&[u8]` (< 8000 bytes)|`varbinary(8000)`|
+/// |`Vec<u8>`/`&[u8]`|`varbinary(max)`|
+/// |`Numeric`|`numeric`/`decimal`|
+/// |`Decimal`|`numeric`/`decimal`|
+/// |`XmlData`|`xml`|
+/// |`NaiveDate` (TDS 7.3 >)|`date`|
+/// |`NaiveTime` (TDS 7.3 >)|`time`|
```

Comment:
What does TDS mean here?

Reply:
Microsoft slang. TDS (Tabular Data Stream) is the protocol, and a bit underneath it's explained `SQL Server 2005` uses the version 7.2, later versions are TDS 7.3 or 7.4. The new date and time types were introduced in 7.3.